.drone.star

```diff
@@ -62,29 +62,32 @@ config = {
         "oCIS-1": {
             "earlyFail": True,
             "skip": False,
-            "featurePaths": [
-                "tests/e2e/cucumber/features/{smoke,journeys}/*.feature",
+            "suites": [
+                "journeys",
+                "smoke",
             ],
         },
         "oCIS-2": {
             "earlyFail": True,
             "skip": False,
-            "featurePaths": [
-                "tests/e2e/cucumber/features/smoke/{spaces,admin-settings}/*.feature",
+            "suites": [
+                "admin-settings",
+                "spaces",
             ],
         },
         "oCIS-3": {
             "earlyFail": True,
             "skip": False,
             "tikaNeeded": True,
-            "featurePaths": [
-                "tests/e2e/cucumber/features/smoke/{search,shares}/*.feature",
+            "suites": [
+                "search",
+                "shares",
             ],
         },
         "oCIS-app-provider": {
             "skip": False,
-            "featurePaths": [
-                "tests/e2e/cucumber/features/smoke/app-provider/*.feature",
+            "suites": [
+                "app-provider",
```

Comment:
not working for single suite. have to fix it

Reply:
fixed: https://github.com/owncloud/web/pull/10557/commits/635a478805467d7186a26a86b225c64ae5d1aa0d